src/frontend/running.rs

```diff
@@ -287,13 +291,22 @@ impl RunningView {
                             backend; qed",
                         ),
                         FarmWidgetInit {
-                            farm,
+                            farm: farm.clone(),
                             total_sectors: initial_farm_state.total_sectors_count,
                             plotted_total_sectors: initial_farm_state.plotted_sectors_count,
                             plotting_paused: self.plotting_paused,
                         },
                     );
+
+                    // Want this to panic in case of any error which I don't expect to happen
+                    space_pledged += farm
+                        .size
+                        .parse::<ByteSize>()
+                        .expect("Failed to parse farm size")
```

Comment:
This is not a proof though. Also we should have actual space pledged on the backend parsed already, I'd probably return it instead of parsing again here, but I didn't check the code before writing this comment.

Reply:
> Also we should have actual space pledged on the backend parsed already
I didn't find any. So, I pulled from frontend as you asked for before.

Reply:
https://github.com/subspace/space-acres/blob/2b4b8a776b56cf1e4fa63ad3e054b689e0cc1043/src/backend/farmer.rs#L176

It is contained in parsed (not raw) config. Previously parsed config was also sent to the frontend, but later removed as unused. You can simply bring it back.